pyfr/plugins/fwh.py

```diff
@@ -67,6 +68,11 @@ def __init__(self, intg, cfgsect, suffix=None, *args, **kwargs):
         self.ele_surface, _ = self._surf_region(intg)
         self._init_surf(ele_map)
 
+        # Get boundary type info
+        sname = self.cfg.get(cfgsect, 'surface')
+        bcsect = f'soln-bcs-{sname}'
+        self.bctype = self.cfg.get(bcsect, 'type', None)
```

Comment:
Is the None needed here?  I think all boundaries have to have a type?

Reply:
Surface may not be a boundary?

Reply:
For this the correct check is `if '(' not in sname:` then you can be sure it is a boundary.